Date: Thu, 1 Nov 2007 21:32:25 +0900
From: Paul Mundt <lethal@linux-sh.org>
To: Andi Kleen <ak@suse.de>
Subject: [PATCH] numactl: Fix bogus __NR_migrate_pages reference.
Message-ID: <20071101123225.GA28035@linux-sh.org>
MIME-Version: 1.0
Content-Type: text/plain; charset=us-ascii
Content-Disposition: inline
User-Agent: Mutt/1.5.13 (2006-08-11)
Status: RO

syscall.c does a bunch of ifdefs to make sure a platform has all of the
syscall numbers defined in the headers or falls back on provided
defaults. Unfortunately __NR_migrate_pages was wrong (it was written as
NR_migratepages), so the common definitions were never picked up.

Signed-off-by: Paul Mundt <lethal@linux-sh.org>

---

 syscall.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- numactl-1.0.2.orig/syscall.c	2007-08-16 23:36:23.000000000 +0900
+++ numactl-1.0.2/syscall.c	2007-11-01 21:22:47.000000000 +0900
@@ -23,7 +23,7 @@
 #define WEAK __attribute__((weak))
 
 #if !defined(__NR_mbind) || !defined(__NR_set_mempolicy) || \
-    !defined(__NR_get_mempolicy) || !defined(NR_migratepages)
+    !defined(__NR_get_mempolicy) || !defined(__NR_migrate_pages)
 
 #if defined(__x86_64__)
 
